README.md

```diff
@@ -5,6 +5,15 @@
 
 # Okta Mobile SDK for Swift
 
+The Okta Mobile SDK replaces our legacy mobile SDKs (such as okta-oidc-ios) and enables:
```

Comment:
Enumerate which legacy mobile SDKs are replaced to avoid confusion? Or maybe qualify the statement in some way so they don't think the DeviceSDK or other swift sdk's are being replaced.

Reply:
Good observation. I originally had a list of SDKs this is replacing, but didn't want to "spook the wildlife" since we don't have release estimates or notices of deprecation yet.  We'll come up with a definitive list a bit later, once we identify a migration plan.